docs/en/operations/settings/settings.md

```diff
@@ -4659,3 +4659,44 @@ The default value is `false`.
 ``` xml
 <validate_tcp_client_information>true</validate_tcp_client_information>
 ```
+
+## ignore_eacces_multidirectory_globs {#ignore_eacces_multidirectory_globs}
```

Comment:
1. Typo: `acces` -> `access`.

2. `eaccess` sounds weird.
I know that it is from Linux's/POSIX's `EACCESS`.
But let's not follow this notation.

`ignore_paths_without_permission_in_multidirectory_globs` will be better.

> This setting is only applicable when multi directory `{}` glob is used.

This is difficult to understand.
I didn't read the changes, didn't read the PR description, but from the docs, it makes me wonder:
- why doesn't it apply for `**` which is also a multi-directory glob;
- why don't we have a similar setting to skip files (not directories) without permissions when using simple globs?

Reply:
> why doesn't it apply for ** which is also a multi-directory glob    

`**` skips them anyway, isn't it?

> why don't we have a similar setting to skip files (not directories) without permissions when using simple globs?

Well, I was just addressing the problem that came with newly introduced feature. Previously, globs worked as is and everyone seemed to be satisfied.
DYM this setting shall be applied to all globs? Or you mean that it shall be explicitly specified in docs that it doesn't work for other settings?

Reply:
Minor:

> Typo: acces -> access

No, not a typo, it is `eacces`

> But let's not follow this notation.

Why? It doesn't really matter to me personally 😄 But `eacces` is a rather conventional name, and it would be easy to find if you know what you're looking for. I don't think `ignore_paths_without_permission_in_multidirectory_globs` is really better -- isn't it a bit too long?
There are also references to this setting from `globs` sections, so it still can be found.

Reply:
Anyway, setting renamed